e2etest/pkg/routes/routes.go

```diff
@@ -0,0 +1,80 @@
+package routes
```

Comment:
I guess a lot of this will get reworked based on @oribon 's good comment about the JSON format. Therefore, I will not review this file.